kpatch-build/create-klp-module.c

```diff
@@ -456,12 +458,12 @@ int main(int argc, char *argv[])
 	ksymsec = find_section_by_name(&kelf->sections, ".kpatch.symbols");
 	if (!ksymsec)
 		ERROR("missing .kpatch.symbols section");
-	ksyms_nr = ksymsec->data->d_size / sizeof(struct kpatch_symbol);
+	ksyms_nr = (int)(ksymsec->data->d_size / sizeof(struct kpatch_symbol));
```

Comment:
Would be more sensible to have ksyms_nr as `unsigned int`.

Reply:
Agree, they can't be negative. Convention, we have been following is to use `int` instead of `unsigned int`. I will leave that to debate. One exception, I will prefer them is using as an index into an array.